performance_regression_test.py

```diff
@@ -891,3 +892,71 @@ def test_latency_write_with_upgrade(self):
     def test_latency_mixed_with_upgrade(self):
         self._prepare_latency_with_upgrade()
         self.run_workload_and_upgrade(stress_cmd=self.params.get('stress_cmd_m'))
+
+
+class PerformanceRegressionMaterializedViewLatencyTest(PerformanceRegressionTest):
+    """
+    the idea is to reproduce the hardest scenario for MV
+    based on internal doc "Consistency problems in materialized views"
+
+    modifying a column that is a regular column in the base table,
+    but in the materialized view is one of the primary key columns.
+    Other types of materialized view updates are easier to handle,
+    once we figure out how to do the hardest case correctly, all of the other cases will be solved as well.
+
+    currently this problem is not solved.
+    The test is just reproducer of this problem and should not be used in regular runs
+
+    test steps:
+    1 - 3 node cluster with 2 tables
+    2 - do special prepare CMD for table 1, and use table 2 as for latency PERF TEST (prepare_write_cmd)
+    3 - start read workload for table 2 - measure latency for table 2 (10min) (stress_cmd_r)
+    4 - do a special rewrite workload for table 1 to measure latency for table 2 (while changing for table 1 applying )(stress_cmd_no_mv)
+    5 - create MV, and wait for MV to sync - measure latency for table 2 (while MV is syncing )
+    6- do special rewrite workload for table 1 again - measure latency for table 2 (while changing for table 1 applying ) (stress_cmd_mv)
+    """
+
+    def test_read_mv_latency(self):
+        self.run_fstrim_on_all_db_nodes()
+        self.preload_data()  # prepare_write_cmd
+        self.wait_no_compactions_running()
+        self.run_fstrim_on_all_db_nodes()
+
+        self.create_test_stats(sub_type="read", append_sub_test_to_name=False, test_index="mv-overloading-latency-read")
+        self.run_stress_thread(stress_cmd=self.params.get('stress_cmd_r'), stress_num=1,
```

Comment:
you don't need the result of this command, do you? it is just background workload?